packages/fiori/src/ShellBarPopover.hbs

```diff
@@ -21,6 +23,7 @@
 		{{#each _hiddenIcons}}
 			<ui5-li
 				data-ui5-external-action-item-id="{{this.refItemid}}"
+				data-ui5-stable="{{this.stableDomRef}}"
```

Comment:
This makes sure that when the menu items are displayed in the overflow menu, that the stableDomRefs are preserved here as well.